spec/sample.spec.js

```diff
@@ -0,0 +1,73 @@
+
+var url = "https://dev.isrd.isi.edu/ermrest", cookie = "ermrest=C6KFIQn2JS37CGovofWnjKfu;";
```

Comment:
can you remove these from the code? these should come from the user environment.